### ISSUE TITLE
Property title from an openapi schema object is not shown

### ISSUE THREAD

**Issue by x12a1f** [testingisdocumenting/znai#1041]
Given the following openapi.yaml :

```yaml
openapi: 3.0.3
info:
  title: TEST
  version: '1'
paths:
  /test:
    get:
      responses:
        default:
          description: "Stuff"
          content:
            application/json:
              schema:
                type: object
                properties:
                  name:
                    type: string
                    title: The title
                    description: The description
```

and using 

`:include-open-api: openapi.yaml {method: "get", path: "/test"}`

in a markdown file

The generated documentation does not show the title, it only shows the description:

![2022-11-27_13-38](https://user-images.githubusercontent.com/1533323/204135675-cb9cd616-9152-453a-9f7b-59463ad364a4.png)

I would expect something like:

The title

The description

**Comment by MykolaGolubyev** [comment]
Oh interesting. Will take a look

**Comment by MykolaGolubyev** [comment]
<img width="465" alt="Screenshot 2022-11-29 at 8 47 09 PM" src="https://user-images.githubusercontent.com/11657860/204687827-e11ce577-b3d7-4618-b04a-3dae322a4a0d.png">

**Comment by x12a1f** [comment]
Awesome, look good! Thanks.

**Comment by MykolaGolubyev** [comment]
this has been released as 1.62: https://testingisdocumenting.org/znai/release-notes/2022#1-62
